### DISCUSSION TITLE
Creating a custom Email backend

### DISCUSSION THREAD

**Discussion by enaut** [coddingtonbear/django-mailbox#294] [Ideas]
I'm wondering - wouldn't it make sense to have a custom email backend for the mailboxes? Right now I can send, but the default Django backend is used, which might not allow sending emails from arbitrary domains…

To fix this, I'd add a model for outgoing servers and a custom mailing backend that selects which credentials and server to use for which address, or something similar. I did not dig too deep yet into how to implement this as I might be completely wrong.

What do you think about implementing such a feature in general?

Or did I misunderstand something completely?

**Comment by Pietro395** [comment]
Hi, 
I think sending emails via this project is out of scope, it would be better to implement a custom email backend in your project following Django's documentation:
https://docs.djangoproject.com/en/5.0/topics/email/#defining-a-custom-email-backend

**Reply by enaut** [reply]
I'll do just that. The reason why I considered adding this to django-mailbox is that I will most probably have foreign keys between my own and django-mailboxes models.
